result_collector/aggregate_results.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import re
+
+def get_number(filename):
+    return re.findall(r'\d+', filename)[0]
```

Comment:
Could this be done without a regular expression? What does filename look like?

Reply:
file names look like xs_tag<number>.npy

Reply:
Yes, probably we can, I just did it in the most straightforward way. Probably we can add one more '_' and split by that

Reply:
How about `os.path.splitext(os.path.basename(filename).split('_'))[0]`? or something like that?